index.html

```diff
@@ -0,0 +1,137 @@
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+        <meta charset="UTF-8">
+        <meta http-equiv="X-UA-Compatible" content="IE=edge">
+        <meta name="viewport" content="width=device-width, initial-scale=1.0">
+        <title>InnovationX Africa Summit</title>
+        <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.css"/>
+        <link rel="stylesheet" href="./style/style.css">     
+    </head>
+    <body>
+      <section class="container">
+        <header>
+          <div class="top-navigation">
+            <div class="media-links">
+              <ul>
+                <li><a><i class="fa fa-facebook" aria-hidden="true"></i></a></li>
+                <li><a><i class="fa fa-twitter" aria-hidden="true"></i></a></li>
+                <li><a>English</a></li>
+                <li><a>My Page</a></li>
+                <li><a>Logout</a></li>
+              </ul>
+            </div>
+            <div>
+              <li class="logo-link"><a href="index.html"><img class="summitlogo" src="./images/logos/CSi.png" alt="Summit Logo"/></a>
+                <ul>
+                  <li><a href="about_page.html" id="about-active"> About</a></li>
+                  <li><a href="#"> Program</a></li>
+                  <li><a href="#"> Join</a></li>
+                  <li><a href="#"> Sponsor</a></li>
+                  <li><a href="#"> News</a></li>
+                  <li class="campaign"><a href="#"> IX Campaign</a></li>
+                </ul>
+              </div> 
```

Comment:
- Hey, in here also, as this is the second nav tag and not a div, to also enforce good use of the semantic tag, the container for this part of your webpage should also be a nav tag as this is the second nav tag on your page that contains the list of the main page navigation links. let's improve on this. :pray: